nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -289,8 +289,17 @@ waypoint_follower:
   ros__parameters:
     loop_rate: 20
     stop_on_failure: false
-    waypoint_task_executor_plugin: "waypoint_task_executor"   
-    waypoint_task_executor:
+    # Choose a plugin from provided ones; 
+    # photo_at_waypoint or photo_at_waypoint ?
+    waypoint_task_executor_plugin: "photo_at_waypoint"   
```

Comment:
Default should be wait at waypoint, this is a specific application.

Reply:
So I think revert all these changes in the yaml file

Reply:
Here's a good question: Should we support multiple plugins at a waypoint? So rather than just 1 option, can we have a vector of them like the controllers / planners?

Reply:
thought about this before but then also thought , we can just define sequential tasks to be executed inside `processAtWaypoint() ` function itself. but if tasks are more complex and irrelevant a vector approach would be much more useful here. I can work towards that but I want to clean up the plugin itself first and make it camera ready for merge. After that it will be easy to modify waypoint_follower such that its accepts multi plugins

Reply:
OK